.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Description
+*Please describe what changes you made, and why you feel they are necessary. Make sure to include
+code examples, where applicable.*
+
+Corresponding issue:
+
+## Testing
+*Please explain how you tested this change manually, and, if applicable, what new tests you added. If
+you're making a change to just the website, you can omit this section.*
+
+## Checklist
+
+* [ ] I have read the contribution guidelines
+* [ ] I have targeted this PR against the correct branch (`master` for website changes, `dev` for
+  source changes)
+* [ ] This is either a bugfix, a documentation update, or a new feature that has been explicitly
+  approved via an issue
+* [ ] I ran the test suite locally (`npm run test`) and verified that it succeeded
```

Comment:
I would change this point to:

>I added test cases that back my changes and run it locally (`npm run test`)

Reply:
There are situations in which new tests aren't required, so all I want to know is that they've run the test suite.